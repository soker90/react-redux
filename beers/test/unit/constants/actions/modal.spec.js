// modal.spec.js
import modalConstants from '../../../../src/constants/actions/modal'

describe('Modal constants', function () {
  test('exist', () => {
    expect(modalConstants).toMatchSnapshot()
  })
})
