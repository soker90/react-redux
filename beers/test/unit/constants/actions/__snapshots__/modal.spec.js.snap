exports[`Modal constants exist 1`] = `
Object {
  "CLOSE_MODAL": "CLOSE_MODAL",
  "OPEN_MODAL": "OPEN_MODAL",
}
`;
